codecov_auth/migrations/0001_initial.py

```diff
@@ -6,7 +6,7 @@
 import django.contrib.postgres.fields
 import django.contrib.postgres.fields.citext
 import django.db.models.deletion
-from django.conf import settings
+from django.conf import settings  # noqa: F401
```

Comment:
I didnt want to mess with migrations so added these back after auto fix and ignored